.gitignore

```diff
@@ -90,3 +90,5 @@ ENV/
 
 # IDE
 .idea/
+
+*.json
```

Comment:
I don't think we should add this. If we add more example equipment responses in the future is going to just make it harder to get them added in the commits.